thoth/storages/graph/models.py

```diff
@@ -1499,7 +1499,8 @@ class PythonPackageMetadataProjectUrl(Base, BaseExtension):
     __tablename__ = "python_package_metadata_project_url"
 
     id = Column(Integer, primary_key=True, autoincrement=True)
-    project_url = Column(Text, nullable=True)
+    url = Column(Text, nullable=True)
+    source = Column(Text, nullable=True)
```

Comment:
It might be a good idea to name this as a [label to follow the specification](https://packaging.python.org/en/latest/specifications/core-metadata/#project-url-multiple-use).

Reply:
Thanks :100: Changes were added in https://github.com/thoth-station/storages/pull/2568/commits/7260f0b9cdf29babb02a42fe25161a2095930253